src/sentry/utils/avatar.py

```diff
@@ -88,3 +91,17 @@ def get_letter_avatar(display_name, identifier, size=None, use_svg=True):
             '{initials}</span>').format(color=color, initials=initials,
                                         size_attrs=size_attrs, font_size=font_size,
                                         line_height=line_height)
+
+
+def get_email_avatar(display_name, identifier, size=None):
+    try:
+        validate_email(identifier)
```

Comment:
not sure if it's worth doing this or not. just figured it was one way to not even bother trying to get a gravatar if we don't have an email address.